packages/ai-chat/src/chat/react/carbon/DatePicker.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright IBM Corp. 2025
+ *
+ *  This source code is licensed under the Apache-2.0 license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ *  @license
+ */
+
+import { createComponent } from "@lit/react";
+import React from "react";
+
+// Export the actual class for the component that will *directly* be wrapped with React.
+import CarbonDatePickerElement from "@carbon/web-components/es-custom/components/date-picker/date-picker.js";
+import CarbonDatePickerInputElement from "@carbon/web-components/es-custom/components/date-picker/date-picker-input.js";
```

Comment:
We probably want to import from `es` instead of `es-custom` to align with this PR that will come in later: https://github.com/carbon-design-system/carbon-ai-chat/pull/367